packages/core-ethereum/src/channel.ts

```diff
@@ -77,6 +77,10 @@ class Channel {
     }
   }
 
+  async initCommitment() {
```

Comment:
I would call it 'bumpCommitment'

Reply:
why do we want to bump the commitment on init? This seems wrong.

Reply:
obsolete since method is renamed